Assets/MixedRealityToolkit/AudioManager/ActiveParameter.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.Audio
+{
+    /// <summary>
+    /// The runtime instance of an AudioParameter on an ActiveEvent
+    /// </summary>
+    public class ActiveParameter : IDisposable
```

Comment:
what's the purpose of the disposable here?

Doesn't seem like we're managing any memory or anything.

Reply:
This was an internal suggestion, and relic from the previous audio manager to avoid garbage collection on the ActiveEvents that are constantly being created and destroyed as sounds are played.

Reply:
Interesting.

Reply:
>  to avoid garbage collection

Did it make a noticeable performance difference?

Reply:
Honestly, I never noticed a performance difference, it seemed like a premature optimization, so I'm totally fine with it either way.

Reply:
Let's go without IDisposable. If we find a perf/garbage issue, we can address then :)